### PR TITLE
[PL-1023] - Prospective storyscripter sees architecture diagram for example story

### PR DESCRIPTION
**Pull request by Arinono**
As a prospective storyscripter,
When I navigate to the playground counter example,
I see a topology diagram showing the interacting components of the example story,
So that I understand the platform building blocks

## Acceptance
**Given** I navigate to the playground,
**Then** I see the topology diagram for the example story including the http and redis services

## Notes
Screenshot 2019-11-04 at 18 27 16

### CODE REVIEW

File: src/samples/counter.ts [hunk 1]
```diff
@@ -2,43 +2,55 @@ import { IStorySample } from '@/models/StorySample'
...
-      'redis'
-    ]
-  },
+  services: [
+    { icon: 'redis.svg', name: 'http' },
+    { icon: 'redis.svg', name: 'redis' }
+  ],
+  architecture: [
+    [{ icon: 'redis', health: 1 }, { icon: 'python', health: -1 }],
+    [{ icon: 'redis', health: 1 }, { icon: 'python', health: -1 }],
```
Comment (wilzbach) [inline src/samples/counter.ts]: Uhm I presume this duplication is just for testing purposes?

File: src/samples/counter.ts [hunk 2]
```diff
@@ -2,43 +2,55 @@ import { IStorySample } from '@/models/StorySample'
...
-    files: [
-      'counter'
-    ],
-    services: [
-      'http',
-      'redis'
-    ]
-  },
+  services: [
+    { icon: 'redis.svg', name: 'http' },
```
Comment (wilzbach) [inline src/samples/counter.ts]: Shouldn't this point to an http icon?

File: src/samples/counter.ts (hunk 1 above)
Comment (Arinono) [reply src/samples/counter.ts]: Yes, I need to ask Ayda for some missing icons (nginx, k8s, ...)

File: src/samples/counter.ts (hunk 2 above)
Comment (Arinono) [reply src/samples/counter.ts]: Yes, I need to ask Ayda for some missing icons (nginx, k8s, ...)

**Comment by williammartin** [comment]
### Acceptance

**Given** I navigate to the playground,
**Then** I see the architecture diagram for the example story including the http and redis services

image

LGTM thanks

**Comment by williammartin** [comment]
We can fix the blurry kubernetes logo in another issue.

**Pull request merged**